### ISSUE TITLE
AssertionError: Multiple .dist-info directories: pandaSDMX-0.8 and 0.7.0

### ISSUE THREAD

**Issue by Xparx** [dr-leo/pandaSDMX#60]
While trying to install this package I ran in to the issue in the title, full traceback here:

```
Exception:
Traceback (most recent call last):
  File "~/.virtualenvs/default/lib/python3.5/site-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "~/.virtualenvs/default/lib/python3.5/site-packages/pip/commands/install.py", line 342, in run
    prefix=options.prefix_path,
  File "~/.virtualenvs/default/lib/python3.5/site-packages/pip/req/req_set.py", line 784, in install
    **kwargs
  File "~/.virtualenvs/default/lib/python3.5/site-packages/pip/req/req_install.py", line 851, in install
    self.move_wheel_files(self.source_dir, root=root, prefix=prefix)
  File "~/.virtualenvs/default/lib/python3.5/site-packages/pip/req/req_install.py", line 1064, in move_wheel_files
    isolated=self.isolated,
  File "~/.virtualenvs/default/lib/python3.5/site-packages/pip/wheel.py", line 345, in move_wheel_files
    clobber(source, lib_dir, True)
  File "~/.virtualenvs/default/lib/python3.5/site-packages/pip/wheel.py", line 305, in clobber
    ', '.join(info_dir))
AssertionError: Multiple .dist-info directories: ~/.virtualenvs/default/lib/python3.5/site-packages/pandaSDMX-0.8.dist-info, ~/.virtualenvs/default/lib/python3.5/site-packages/pandaSDMX-0.7.0.dist-info
```

To solve it I had to specify the version in pip to make installing work (version 0.8 didn't work),
```
pip install pandasSDMX==0.7.0
```
worked as far as I can tell.

**Comment by dr-leo** [comment]
Hi,

this is a known packaging bug probably caused by the Wheel tool. I'll
do a new release asap.

Meanwhile you may try to download the 0.8 tarball manually and install
with python Setup.py install.

Leo

**Comment by Xparx** [comment]
Ah, ok. I somehow missed this was a know error from the sources I checked.
I will try installing from the tarball. thanks for the help!

**Comment by dr-leo** [comment]
Fixed in release v0.8.1
